crt_portal/cts_forms/model_variables.py

```diff
@@ -12,21 +12,39 @@
 )
 
 PROTECTED_CLASS_CHOICES = (
-    ('race_color', 'Race/color'),
-    ('national_origin', 'National origin (including ancestry, ethnicity, and language)'),
-    ('immigration_citizenship', 'Immigration or citizenship status (you do not have to divulge your status)'),
+    'Disability (including temporary or in recovery)',
+    'Race/color',
+    'National origin (including ancestry, ethnicity, and language)',
+    'Immigration/citizenship status (choosing this does not share your status)',
+    'Religion',
+    'Sex or gender identity (including gender stereotypes) or pregnancy',
+    'Sexual orientation',
+    'Family, marriage, or parental status',
+    'Military status',
+    'Age',
+    'Genetic information',
+    'Other',
+)
+
+PROTECTED_MODEL_CHOICES = (
```

Comment:
@ilodidoj and I had a couple of questions here:

* What is the difference between `PROTECTED_CLASS_CHOICES` and `PROTECTED_MODEL_CHOICES`?
* Is there ever a case where we'd want the second tuple item in `PROTECTED_MODEL_CHOICES` to not match `PROTECTED_CLASS_CHOICES`?
* If they should always match, should we derive the values in one constant from the values in the other constant to make sure they are a guaranteed match?

Reply:
@ilodidoj and @alexsoble  The `PROTECTED_CLASS_CHOICES` determine what is populated in the form. `PROTECTED_MODEL_CHOICES` determine what is allowable data in the models. 

For example, let's say there is a new need to track national origin and ethnicity separately. Now we just track "National origin (including ancestry, ethnicity, and language)".

So for the old record, we don't want to change them, because we don't know how people replied to the form, but going forward, we don't want to give "National origin (including ancestry, ethnicity, and language)" as an option on the form anymore. 

To do that, we would keep that and add the new classes "National origin" and "Identity" in `PROTECTED_MODEL_CHOICES` and leave the original option. That way, we don't change any of the old data. To make sure the public facing options are correct, we would remove "National origin (including ancestry, ethnicity, and language)" and add "National origin" and "Identity" in `PROTECTED_CLASS_CHOICES`. We would then remove the form order from National origin (including ancestry, ethnicity, and language) and add the correct order to the new elements. We would then make sure that the "Other"javascript was pointed at the correct element.

Reply:
Ooooooh, thanks, this is a great explanation! Can this be a code comment?

Reply:
Ok, that is a good idea. 

I added a slightly longer comment in our `/docs` and put a bread crumb next to the protected class model, and the variable definitions